doc/releases/changelog-dev.md

```diff
@@ -4,92 +4,127 @@
-  2j * Z(0) @ X(1)
-  + 2j * X(0) @ Z(1)
+  @qml.qnode(dev)
+  def f():
+      for i in range(1967):
+          qml.Hadamard(0)
+          qml.measure(0)
+      return qml.sample(qml.PauliX(0))
   ```
-
-  We can also compute commutators with Pauli operators natively with the `PauliSentence.commutator` method.
-
   ```pycon
-  >>> op1 = PauliWord({0:"X", 1:"X"})
-  >>> op2 = PauliWord({0:"Y"}) + PauliWord({1:"Y"})
-  >>> op1.commutator(op2)
-  2j * Z(0) @ X(1)
-  + 2j * X(0) @ Z(1)
+  >>> f()
+  tensor([-1, -1, -1,  1,  1, -1,  1, -1,  1, -1], requires_grad=True)
   ```
 
-* Upgrade Pauli arithmetic:
-  You can now multiply `PauliWord` and `PauliSentence` instances by scalars, e.g. `0.5 * PauliWord({0:"X"})` or `0.5 * PauliSentence({PauliWord({0:"X"}): 1.})`.
-  You can now intuitively add together
-  `PauliWord` and `PauliSentence` as well as scalars, which are treated implicitly as identities.
-  For example `ps1 + pw1 + 1.` for some Pauli word `pw1 = PauliWord({0: "X", 1: "Y"})` and Pauli
-  sentence `ps1 = PauliSentence({pw1: 3.})`.
-  You can now subtract `PauliWord` and `PauliSentence` instances, as well as scalars, from each other. For example `ps1 - pw1 - 1`.
-  Overall, you can now intuitively construct `PauliSentence` operators like `0.5 * pw1 - 1.5 * ps1 + 2`.
-  You can now also use `qml.dot` with `PauliWord`, `PauliSentence` and operators, e.g. `qml.dot([0.5, -1.5, 2], [pw1, ps1, id_word])` with `id_word = PauliWord({})`.
-  [(#4989)](https://github.com/PennyLaneAI/pennylane/pull/4989)
-  [(#5001)](https://github.com/PennyLaneAI/pennylane/pull/5001)
-  [(#5003)](https://github.com/PennyLaneAI/pennylane/pull/5003)
-  [(#5017)](https://github.com/PennyLaneAI/pennylane/pull/5017)
-  [(#5027)](https://github.com/PennyLaneAI/pennylane/pull/5027)
-
-* `qml.matrix` now accepts `PauliWord` and `PauliSentence` instances, `qml.matrix(PauliWord({0:"X"}))`.
-  [(#5018)](https://github.com/PennyLaneAI/pennylane/pull/5018)
-
-* Composite operations (eg. those made with `qml.prod` and `qml.sum`) and `SProd` operations convert `Hamiltonian` and
-  `Tensor` operands to `Sum` and `Prod` types, respectively. This helps avoid the mixing of
-  incompatible operator types.
-  [(#5031)](https://github.com/PennyLaneAI/pennylane/pull/5031)
-  [(#5063)](https://github.com/PennyLaneAI/pennylane/pull/5063)
-
-* `qml.Identity()` can be initialized without wires. Measuring it is currently not possible though.
-  [(#5106)](https://github.com/PennyLaneAI/pennylane/pull/5106)
-
-<h4>Easy to inspect transforms 🔎</h4>
-
-<h4>New Clifford and noisy qutrit devices 🦾</h4>
+  Previously, mid-circuit measurements would be automatically replaced with an additional qubit
+  using the `@qml.defer_measurements` transform, so the above circuit would have required thousands
+  of qubits to simulate.
+
+<h4>Work easily and efficiently with operators 🔧</h4>
+
+* Over the past few releases, PennyLane's approach to operator arithmetic has been overhauled.
```

Comment:
I think the first release for new operator arithmetic was the birth of triangle head guy lol